update-eks-kubeconfig/update-eks-kubeconfig.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Strict Mode - http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+#URL to download the kubeconfigs
+BUCKET_URL=https://upp-kubeconfig.s3-eu-west-1.amazonaws.com
+
+# Check if the bucket is reachable
+status=$(curl -s --head -w %{http_code} $BUCKET_URL/index -o /dev/null)
+if [ $status -ne "200" ] 
+then
+  echo "S3 bucket not reachable. Check your connection"
+  exit 1
+fi
+
+# Get kubeconfigs
+for config in $(curl -s $BUCKET_URL/index); do
+curl -s $BUCKET_URL/$config > $config
+done
+
+#Merge kubeconfigs
+KUBECONFIG=$(curl -s $BUCKET_URL/index | tr "\n" ':' | sed 's/:$//g') kubectl config view --merge=true --flatten=true > /tmp/kubeconfig
```

Comment:
- Why `--flatten=true`? I think we don't need to flatten the file.
- `tr "\n" ':' | sed 's/:$//g'` Adding `:` and then removing the last one? Try something like this: https://stackoverflow.com/questions/11393817/read-lines-from-a-file-into-a-bash-array or https://github.com/corneliusweig/konfig/blob/master/konfig#L72-L84

Reply:
- flattening will include certificate-authority-data and will make the file more self contained and complete
- I find this way more simple and explanatory then compiling an array.

Reply:
Anyway I have switched to array